### PR TITLE
Add social network to serializer

### PR DESCRIPTION
**Pull request by bderusha**

### CODE REVIEW

File: course_discovery/apps/api/serializers.py [hunk 1]
```diff
@@ -230,19 +234,53 @@ class Meta(object):
     def validate(self, data):
         validated_data = super(PersonSerializer, self).validate(data)
         validated_data['works'] = self.initial_data.get('works', [])
+        validated_data['facebook_url'] = self.initial_data.get('facebook_url')
```
Comment (clintonb) [inline course_discovery/apps/api/serializers.py]: A few for-loops in these methods might help with duplication.

File: course_discovery/apps/api/serializers.py [hunk 2]
```diff
@@ -212,6 +212,10 @@ class PersonSerializer(serializers.ModelSerializer):
     position = PositionSerializer(required=False)
     profile_image = StdImageSerializerField(required=False)
     works = serializers.SlugRelatedField(many=True, read_only=True, slug_field='value', source='person_works')
+    facebook_url = serializers.SerializerMethodField()
```
Comment (clintonb) [inline course_discovery/apps/api/serializers.py]: Consider nesting the social networks:

```json
{
  "name": "John Doe",
  "urls": {
    "facebook": "https://facebook.com/john.doe",
    "twitter": "https://twitter.com/john.doe"
  }
}
```

File: course_discovery/apps/api/serializers.py [hunk 3]
```diff
@@ -212,6 +212,10 @@ class PersonSerializer(serializers.ModelSerializer):
...
+    twitter_url = serializers.SerializerMethodField()
+    blog_url = serializers.SerializerMethodField()
+    other_url = serializers.SerializerMethodField()
```
Comment (clintonb) [inline course_discovery/apps/api/serializers.py]: If we don't have a definition of "other", it shouldn't be in our models/API. The lack of clarity will probably hurt us down the road.

**Comment by bderusha** [comment]
closing in favor of https://github.com/edx/course-discovery/pull/635

**Pull request closed without merging**